guides/common/modules/proc_republishing-repository-metadata.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Republishing_Repository_Metadata_{context}"]
+= Republishing Repository Metadata
+
+Republishing repository metadata is used when a repository distribution does not have the content that should be distributed based on the contents of the repository.
+
+Use this procedure with caution.
+In most cases, {Team} recommends a complete repository sync or publishing a new Content View version.
```

Comment:
I am unsure about the wording "in most cases". Do we know specifically _when_ it's a good idea to republish the metadata vs syncing and republishing a CV?

Maybe a generic "Use to repair broken metadata..."?

Reply:
Going with `{Team} recommends a complete repository sync or publishing a new Content View version to repair broken metadata.`